llvm/lib/Target/AArch64/SVEInstrFormats.td

```diff
@@ -10007,6 +10007,16 @@ multiclass sve2p1_dupq<string mnemonic> {
     bits<1> index;
     let Inst{20} = index;
   }
+
+  def : SVE_2_Op_Imm_Pat<nxv16i8, Op, nxv16i8, i32, VectorIndexB32b_timm, !cast<Instruction>(NAME # _B)>;
```

Comment:
Can you change the vector index in lines 9994, 9998, 10002, 10006?

Reply:
You mean changing them to 64 bits ones?

Reply:
Change them to `_timm`.

Reply:
DOne